### PR TITLE
:bug: Source MSSQL: fix CDC data type decimal and money

### PR DESCRIPTION
**Pull request by duytp**
## What
*Describe what the change is solving*
*It helps to add screenshots if it affects the frontend.*

https://github.com/airbytehq/airbyte/issues/15780

## How
*Describe the solution*
Add Datatype: SMALLMONEY, DECIMAL, NUMERIC in MSSQLConverter.java

### CODE REVIEW

**Comment by duytp** [comment]
@gsmith-schlesinger thank you for your contribution.

**Comment by marcosmarxm** [comment]
Thanks @duytp team will review later today or next week your contribution! Sorry the delay

**Comment by marcosmarxm** [comment]
@subodh1810 can you review this contribution?

File: airbyte-integrations/bases/debezium-v1-4-2/src/main/java/io/airbyte/integrations/debezium/internals/MSSQLConverter.java [hunk 1]
```diff
@@ -165,6 +168,23 @@ private void registerMoney(final RelationalColumn field,
...
 
+  private void registerDecimal(final RelationalColumn field,
+                             final ConverterRegistration<SchemaBuilder> registration) {
+    registration.register(SchemaBuilder.float64(), input -> {
+      if (Objects.isNull(input)) {
+        return DebeziumConverterUtils.convertDefaultValue(field);
+      }
+
+      if (input instanceof BigDecimal) {
+        return ((BigDecimal) input).doubleValue();
```
Comment (subodh1810) [inline airbyte-integrations/bases/debezium-v1-4-2/src/main/java/io/airbyte/integrations/debezium/internals/MSSQLConverter.java:179]: I dont understand why are we converting to the double value? Whats wrong with using BigDecimal as it is? Wouldnt it result in loss of precision and scale?

**Review by subodh1810 — CHANGES_REQUESTED** [review]
The test is failing, we need to add new tests for this change as well

**Comment by marcosmarxm** [comment]
@duytp did you check Subbodh's comments?

**Comment by marcosmarxm** [comment]
Closing due inactivity @duytp let me know if you want to continue this contribution then I can reopen it.

**Pull request closed without merging**